ftw/usermigration/browser/migration.py

```diff
@@ -96,34 +102,39 @@ def handleMigrate(self, action):
             self.status = self.formErrorsMessage
             return
 
-        userids = {}
-        for line in data['user_mapping']:
+        principal_ids = {}
+        for line in data['principal_mapping']:
             try:
-                old_userid, new_userid = line.split(':')
+                old_id, new_id = line.split(':')
             except ValueError:
                 raise WidgetActionExecutionError(
-                    'user_mapping', Invalid('Invalid user mapping provided.'))
-            userids[old_userid] = new_userid
+                    'principal_mapping',
+                    Invalid('Invalid principal mapping provided.'))
+            principal_ids[old_id] = new_id
 
         if 'users' in data['migrations']:
             self.results_users = migrate_users(
-                context, userids, mode=data['mode'], replace=data['replace'])
+                context, principal_ids, mode=data['mode'],
+                replace=data['replace'])
 
         if 'properties' in data['migrations']:
             self.results_properties = migrate_properties(
-                context, userids, mode=data['mode'], replace=data['replace'])
+                context, principal_ids, mode=data['mode'],
+                replace=data['replace'])
 
         if 'dashboard' in data['migrations']:
             self.results_dashboard = migrate_dashboards(
```

Comment:
@lukasgraf Does the `migrate_dashboards` and the  function handles group ids properly (ignore)?